pristine-zen-garden.html

```diff
@@ -0,0 +1,157 @@
+  <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+  <meta name="robots" content="all">
+
+</head>
+
+<body id="css-zen-garden">
+  <div class="page-wrapper">
+
+    <section class="intro" id="zen-intro">
+      <header role="banner">
+        <h1>CSS Zen Garden</h1>
+        <h2>The Beauty of <abbr title="Cascading Style Sheets">CSS</abbr> Design</h2>
+      </header>
+
+      <div class="summary" id="zen-summary" role="article">
+        <p>A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets">CSS</abbr>-based design. Select any style sheet from the list to load it into this page.</p>
+        <p>Download the example <a href="/examples/index" title="This page's source HTML code, not to be modified.">html file</a> and <a href="/examples/style.css" title="This page's sample CSS, the file you may modify.">css file</a></p>
+      </div>
+      <div class="preamble" id="zen-preamble" role="article">
+        <h3>The Road to Enlightenment</h3>
+        <p>Littering a dark and dreary road lay the past relics of browser-specific tags, incompatible <abbr title="Document Object Model">DOM</abbr>s, broken <abbr title="Cascading Style Sheets">CSS</abbr> support, and abandoned browsers.</p>
+        <p>We must clear the mind of the past. Web enlightenment has been achieved thanks to the tireless efforts of folk like the <abbr title="World Wide Web Consortium">W3C</abbr>, <abbr title="Web Standards Project">WaSP</abbr>, and the major browser creators.</p>
+        <p>The CSS Zen Garden invites you to relax and meditate on the important lessons of the masters. Begin to see with clarity. Learn to use the time-honored techniques in new and invigorating fashion. Become one with the web.</p>
+      </div>
+
+
+    </section>
+    <div class="main supporting" id="zen-supporting" role="main">
+      <div class="explanation" id="zen-explanation" role="article">
+        <h3>So What is This About?</h3>
+        <p>There is a continuing need to show the power of <abbr title="Cascading Style Sheets">CSS</abbr>. The Zen Garden aims to excite, inspire, and encourage participation. To begin, view some of the existing designs in the list. Clicking on any one will load the style sheet into this very page. The <abbr title="HyperText Markup Language">HTML</abbr> remains the same, the only thing that has changed is the external <abbr title="Cascading Style Sheets">CSS</abbr> file. Yes, really.</p>
+        <p><abbr title="Cascading Style Sheets">CSS</abbr> allows complete and total control over the style of a hypertext document. The only way this can be illustrated in a way that gets people excited is by demonstrating what it can truly be, once the reins are placed in the hands of those able to create beauty from structure. Designers and coders alike have contributed to the beauty of the web; we can always push it further.</p>
+      </div>
```

Comment:
Hey Priya, this looks better than mine but isn't there a </div> missing here? Please let me know when this issue is resolved.

Reply:
Hi @Hameet , Thank you for your note but saying that I didn't quite understand what you mean by "Isn't there missing here??" Can you please give me bit more description  of "missing".

Reply:
Oh crap! Been typing away all day long, forgot to read what I wrote. I think I meant a  closing </div> tag is missing.

Reply:
Hi @Hameet, Just checked the code through the validator again. If you are thinking about the closing tag for  the div then it  closes on line 74. 
There is no error so far in the code according to  the  HTML validator. Hope it helps,
and yes completely understand writing the code whole day :)

Reply:
There's a validator? On Github? Could you please post the link. Thanks.